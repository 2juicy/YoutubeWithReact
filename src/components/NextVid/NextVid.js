import React from "react";
import "./NextVid.scss";
import { Checkbox, Divider } from "semantic-ui-react";
import VidPreview from "../VidPreview/VidPreview";

export default function NextVid() {
  return (
    <>
      <div className="next-vid-container">
        <h4>Up next</h4>
        <div className="up-next-toggle">
          <span>Autoplay</span>
          <Checkbox toggle defaultChecked />
        </div>
      </div>
      <VidPreview horizontal />
      <Divider />
    </>
  );
}
