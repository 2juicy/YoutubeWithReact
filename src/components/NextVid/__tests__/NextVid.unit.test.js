import React from "react";
import { shallow } from "enzyme";
import NextVid from "../NextVid";

describe("NextVid", () => {
  test("renders", () => {
    const wrapper = shallow(<NextVid />);
    expect(wrapper).toMatchSnapshot();
  });
});
