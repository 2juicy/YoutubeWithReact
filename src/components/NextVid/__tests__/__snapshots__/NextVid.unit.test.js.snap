// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NextVid renders 1`] = `
<Fragment>
  <div
    className="next-vid-container"
  >
    <h4>
      Up next
    </h4>
    <div
      className="up-next-toggle"
    >
      <span>
        Autoplay
      </span>
      <Checkbox
        defaultChecked={true}
        toggle={true}
        type="checkbox"
      />
    </div>
  </div>
  <VidPreview
    horizontal={true}
  />
  <Divider />
</Fragment>
`;
